Dockerfile

```diff
@@ -46,10 +46,8 @@ RUN Rscript -e "\
   print(pkg_deps); \
   pak::pak(pkg_deps); \
   "
-
-COPY . /workflow.scenario.preparation
-
-WORKDIR /workflow.scenario.preparation
```

Comment:
Specifying `WORKDIR` should probably stay in.

Reply:
Also, good practice to specify `USER` so the script doesn't run as root.

Reply:
How should I do this?